doc/ExpertOptions.md

```diff
@@ -1254,33 +1268,23 @@ By default, temporary workspaces hold the results of executed workflows and appl
 }
 ```
 
-This will be passed down through the entire workflow, sub-workflows, and tasks. Workspaces will remain intact for 72 hours. This is a runtime flag, so you will need to run the toplevel workflow with that flag:
+This will be applied to the top-level workflow, sub-workflows, and tasks during compilation.
+
+However, since it is a runtime option, compiling your workflow with this attribute set to true will only guarantee the workspaces of subjobs (that sprawn from the parent job) to remain intact for around 3 days after the analysis. 
```

Comment:
```suggestion
However, since it is a runtime option, compiling your workflow with this attribute set to true will only guarantee the workspaces of subjobs (that sprawn from the parent jobs, which correspond to tasks) to remain intact after the analysis. 
```

And I'd mention the time somewhere above, e.g. at the top:

By default, temporary workspaces hold the results of executed workflows and applets. Normally, these are garbage collected by the system. If you wish to leave them around longer (around 3 days) for debugging purposes, please use: